go.mod

```diff
@@ -6,6 +6,7 @@ module github.com/hyperledger/aries-framework-go
 
 require (
 	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
+	github.com/go-openapi/runtime v0.19.3
```

Comment:
As a followup, it would be good to see if we could limit this dependency to the cmd package. Created https://github.com/hyperledger/aries-framework-go/issues/155